v2.4/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -226,18 +203,17 @@ EOF
 Notice that we've specified an order of 500.  This means that this policy will be applied before any
 of the Kubernetes policies.
 
-We also need to create a policy which allows traffic to access kube-dns.  Let's create a policy which
-applies to the entire cluster, and allows it to access kube-dns pods in the kube-system Namespace.
+We also need to create a policy which allows traffic to access kube-dns.  Let's create a policy that allows it to access kube-dns pods in the kube-system Namespace.
 We'll specify an order of 400 so that it takes precendent over other policies.
 
 ```
-calicoctl apply -f - <<EOF
+$ calicoctl apply -f - <<EOF
```

Comment:
This was done to match the other code snippets around it, happy to remove it and maybe the others?